lib/controller.js

```diff
@@ -599,6 +599,27 @@ controller.updateTesselWithVersion = function(force, tessel, currentVersion, bui
     });
 };
 
+controller.tesselFirmwareVerion = function(opts) {
+  return Tessel.get(opts)
+    .then(function(tessel) {
+      return tessel.fetchCurrentBuildInfo()
+        .then(function(versionSha) {
+          updates.requestBuildList().then(function(builds) {
+            // Figure out what commit SHA is running on it
+            var version = updates.findBuild(builds, 'sha', versionSha).version;
+            logs.info('Tessel version: ' + version);
```

Comment:
I suggest we display the name of the T2 whose version we're displaying. This is helpful for folks who might have several in a room and need to know if they accidentally fetched the version of the wrong one. You can access the name with `tessel.name`.